bundle/edu.gemini.shared.util/src/main/java/edu/gemini/shared/util/HourMinuteFormat.java

```diff
@@ -0,0 +1,58 @@
+package edu.gemini.shared.util;
+
+import java.text.FieldPosition;
+import java.text.Format;
+import java.text.ParsePosition;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class HourMinuteFormat extends Format {
```

Comment:
Bryan and Andy stated how they wanted the file format to look, so I just went along with that. 😸 
Keep in mind that `HourMinuteFormat` is some seriously old code... I just moved it and didn't end up using it. But I think having it in `edu.gemini.shared.util` makes a lot more sense than having it in `jsky.app.ot.gemini.editor` and I think when possible, things like this should be shared. Part of the reason we probably have so many implementations of certain things are because they aren't shared and nobody can find them 😺